Samples/WorkflowCombineSampleApp/Podfile

```diff
@@ -0,0 +1,18 @@
+project 'WorkflowCombineSampleApp.xcodeproj'
+platform :ios, '13.0'
+
+target 'WorkflowCombineSampleApp' do
+    pod 'Workflow', path: '../../Workflow.podspec', :testspecs => ['Tests']
+    pod 'WorkflowUI', path: '../../WorkflowUI.podspec', :testspecs => ['Tests']
+    pod 'WorkflowCombine', path: '../../WorkflowCombine.podspec'
+    pod 'WorkflowReactiveSwift', path: '../../WorkflowReactiveSwift.podspec', :testspecs => ['Tests']
```

Comment:
I think it's fine to drop `ReactiveSwift` from the `CombineSampleApp`.

Reply:
I've moved the sample app generation into `Development.podspec` instead